agents/ibm_powervs/fence_ibm_powervs.py

```diff
@@ -237,7 +269,9 @@ def main():
 	all_opt["power_wait"]["default"] = "1"
 	all_opt["stonith_status_sleep"]["default"] = "2"
 	all_opt["api-type"]["default"] = "private"
-	all_opt["proxy"]["default"] = ""
+	# all_opt["proxy"]["default"] = ""
+	# all_opt["method"]["default"] = "onoff"
+	all_opt["method"]["help"] = "-m, --method=[method]          Method to fence (onoff|cycle) (Default: onoff)"
```

Comment:
You can remove the commented out lines here, and `all_opt["method"]["help"]` as well, as that's it's default value.

Reply:
ok, removed both `all_opt["method"]` lines. But activated `all_opt["proxy"]["default"]` again, since it was there before.